src/main/res/layout/imagery_layer_list_item.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="horizontal">
+    <androidx.appcompat.widget.AppCompatRadioButton
+        android:id="@+id/listItemRadioButton"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"/>
+    <ImageButton
+        android:id="@+id/listItemInfo"
+        android:layout_height="wrap_content"
+        android:layout_width="wrap_content"
+        android:layout_alignParentRight="true"
+        android:layout_centerInParent="true"
+        android:background="@android:color/transparent"
+        android:src="?attr/menu_information"
```

Comment:
I suspect ?attr/information_small would look better here (it's a slightly smaller version of the same icon). 

The other thing is that when I tested in an emulator there was some overlap between the text and the (i) in some cases as  the text wasn't wrapping early enough, could you check that?